scompler-kafka.gemspec

```diff
@@ -18,9 +18,9 @@ Gem::Specification.new do |spec|
   end
   spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
-  spec.add_dependency 'activesupport'
   spec.add_dependency 'delivery_boy'
   spec.add_dependency 'dry-configurable'
+  spec.add_development_dependency 'bundler', '~> 1.17'
```

Comment:
We have already updated the version in monolith to version 2.3.11

Reply:
This gem will be removed from the monolith as unused dependency.